test/serializer/abstract/Optional3.js

```diff
@@ -1,4 +1,3 @@
-// Copies of f;:1
```

Comment:
Please explain why this is not longer desirable.

Reply:
My changes broke the test by causing the value of `f` to be derived in the consequent branch of `f && f()`, which counts as a second occurrence of `f;` Since the derivation is legit, I considered updating the comparator to two. But upon digging further I found that I could not reproduce the original problem that you fixed with #1158, even after reverting your change, making me doubt if it was still relevant, so I dropped it. But in hindsight, now, perhaps it is not a bad thing to keep it around as a regression test, so I have added it back.